src/test/java/com/qaprosoft/carina/demo/WebLoginFormTest.java

```diff
@@ -27,11 +27,10 @@ public void testGoodLogIn() {
         homePage.open();
         Assert.assertTrue(homePage.isPageOpened(), "Home page was not opened!");
 
-        homePage.getLogin().openFieldsForLogIn();
-        Assert.assertTrue(homePage.getLogin().isLoginFormPresent(), "Login form was not opened!");
+        homePage.getHeader().getLoginForm().openLoginForm();
```

Comment:
Let's follow next test structure:
public void logicTest() {
PageB pageB = pageA.clickSomeButton();
PageC pageC = pageB.performSomeActions();
It's not clear to understand what will be  displayed after this method call.